frontend/src/__tests__/cypress/cypress/pages/hardwareProfile.ts

```diff
@@ -28,6 +28,16 @@ class HardwareProfileTableToolbar extends Contextual<HTMLElement> {
   }
 }
 
+class HardwareProfileWarningBanner extends Contextual<HTMLElement> {
+  findTitle() {
+    return this.find().find('.pf-v6-c-alert__title');
+  }
+
+  findDescription() {
+    return this.find().find('.pf-v6-c-alert__description');
```

Comment:
```suggestion
    return this.findByTestId()
```
should be less brittle than relying on dom/xpath traversal in patternfly, if we can find/add a `data-test-id`

Reply:
Anyone have insight on how we can add test id's here for title and message

`<Alert
    isInline
    variant="warning"
    title={warningMessages.title}
    data-testid="hardware-profiles-error-alert"
  >
    <p>{warningMessages.message}</p>
  </Alert>`

Reply:
Just realized these classes aren't necessary, will remove them soon